TKGame/Item.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Security.AccessControl;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.VisualBasic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using Microsoft.Xna.Framework.Media;
+
+namespace TKGame
+{
+    class Item : Entity
+    {
+        static Item instance;
+        private static object syncRoot = new object();
+
+        /// <summary>
+        /// lock Item instance
+        /// </summary>
+        public static Item Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    lock (syncRoot)
+                    {
+                        if (instance == null)
+                        {
+                            instance = new Item();
```

Comment:
Same thing with the enemy, this is thread locking which will make it so we can only have one Item instance which will most likely not be the case. Google this Singleton pattern for more details